### ISSUE TITLE
Stackage nighly compilation failure

### ISSUE THREAD

**Issue by alaendle** [freckle/yesod-auth-oauth2#175]
```
 /var/stackage/work/unpack-dir/unpacked/yesod-auth-oauth2-0.7.1.0-09297878eca5acbb7139379aac5ef1a1b7c904fffbc10dfa965315aa37e750aa/src/UnliftIO/Except.hs:12:50: error: [GHC-88464]
           Variable not in scope:
             (<=<)
               :: (Either a0 a1 -> m0 a1)
                  -> (ExceptT e0 m1 a2 -> m1 (Either e0 a2)) -> ExceptT e m a -> m a
           Suggested fix:
             Perhaps use one of these:
               ‘<=’ (imported from Prelude), ‘=<<’ (imported from Prelude)
          |
       12 |       exceptToIO (runInIO . (either throwIO pure <=< runExceptT))
          |                                                  ^^^
```

**Comment by pbrisbin** [comment]
Thanks for the report! I'll look into this now. In the meantime, I have a few basic questions:

- Normally, I have to be careful to watch the commercialhaskell repo to be alerted to this. Sometimes I get at-mentioned in issues there, sometimes not even that. You reporting an issue directly here is perfect! Is that some new process I could perhaps rely on, or are you just going above and beyond?

- I need quite a few `extra-deps` to even resolve a build plan at all against `nightly` (2023-07-29), which makes me surprised that _this_ is what has caused us to get disabled. Am I missing something?

**Comment by pbrisbin** [comment]
Fixed in #176

**Comment by alaendle** [comment]
> Thanks for the report! I'll look into this now. In the meantime, I have a few basic questions:
> 
>     * Normally, I have to be careful to watch the commercialhaskell repo to be alerted to this. Sometimes I get at-mentioned in issues there, sometimes not even that. You reporting an issue directly here is perfect! Is that some new process I could perhaps rely on, or are you just going above and beyond?
> 
>     * ~I need quite a few `extra-deps` to even resolve a build plan at all against `nightly` (2023-07-29), which makes me surprised that _this_ is what has caused us to get disabled. Am I missing something?~ Nevermind! I see 2023-08-01 has all the deps we need now.

No, there is no formal process - please also consider that we all do this voluntarily in our free time. The type of message essentially depends on what kind of problem occurs. Problems with version ranges are usually created directly as an issue on stackage. Compiler errors like here depending on where it fits better (stackage or project repo); so the best advice I could give is to just to check on things from time to time.

**Comment by pbrisbin** [comment]
> No, there is no formal process - please also consider that we all do this voluntarily in our free time

Yes of course, just to be clear: I was in no way insinuating that you all should do more. I appreciate whatever effort is being put in here.

I thought I heard of some automation in the works to help improve the situation of notifying maintainers at some point, so was just wondering if this was the result. Seems not, and that's fine.

**Comment by alaendle** [comment]
Don't worry, we have understood each other correctly 👍 I also can understand the pain, however from a maintainer perspective things are complicated (since a lot of different things can happen that leads to the disabling of a package). And once again thanks for fixing this; i will re-enable `yesid-auth-oauth2` as soon as possible.
